### ISSUE TITLE
Request - REMOVE  Barcode: Series Reserve

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#9571]
Help us understand your request (check below):

- [x] other

**Describe what you're trying to do** 

Clean up unnecessary clutter

[Barcode: Series Reserve](https://arctos.database.museum/info/barcodeseries.cfm) was developed when barcodes were globally unique within Arctos. 

That's no longer true (https://github.com/ArctosDB/arctos/issues/6789), everybody can have a 1 if they want.

The 'claim' process doesn't seem necessary, clearly isn't helping organization (https://github.com/ArctosDB/arctos/issues/9568 is handy, but many more examples exist), and is probably just unnecessary confusion with the current constraints.

Lacking come sort of compelling evidence that this system is useful, I will plan to

* move the "claim" data somewhere accessible, in case anyone wants them
* remove that system/UI
* remove ``is_claimed_barcode`` and rebuild whatever's using it

------------

Please review the [Arctos Code of Conduct](https://arctosdb.org/code-of-conduct/) to ensure respectful communication, and see <https://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html> for more information about submitting and managing Issues.

**Comment by mkoo** [comment]
This seems noncontroversial-- move to roadmap then to dev? It's streamlining a process that's been updated. The roadmap issue will need to have a to-do step of: update doc
